src/components/expression/expression-panel.js

```diff
@@ -224,16 +218,66 @@ genotet.ExpressionPanel.prototype.updateGenes = function(gene) {
       text: gene
     };
   });
-
   var profile = this.container.find('#profile select').empty();
-  this.selectProfiles_ = profile.select2({
-    data: genes,
-    multiple: true
-  });
-  this.container.find('#profile .select2-container').css({
-    width: '100%'
-  });
-  this.updateGeneSelections();
+
+  if (!genes.length) {
+    return;
+  }
+
+  $.fn.select2.amd.require(['select2/data/array', 'select2/utils'],
+    function(ArrayData, Utils) {
+      var CustomData = function($element, options) {
```

Comment:
$element is according to the [select2 document](https://select2.github.io/announcements-4.0.html). I think it is consistent with the source code of select2 and we should keep it.